apps/end-tenancy/controllers/loop.js

```diff
@@ -244,10 +244,13 @@ module.exports = class LoopController extends DateController {
       undefined :
       hoganRender(conditionalTranslate(`pages.${pagePath}.intro`, req.translate), res.locals);
 
+    const outro = conditionalTranslate(`pages.${pagePath}.outro`, req.translate);
+
```

Comment:
by default `req.translate` will return the string key passed in if lookup fails, conditionalTranslate is a wrapper which returns `null`